tests/Unit/Domain/CoordinateMaps/Test_Frustum.cpp

```diff
@@ -56,7 +56,7 @@ void test_suite_for_frustum(const bool with_equiangular_map) {
          {{upper_x_lower_base, upper_y_lower_base}},
          {{lower_x_upper_base, lower_y_upper_base}},
          {{upper_x_upper_base, upper_y_upper_base}}}};
-    const CoordinateMaps::Frustum frustum_map(face_vertices, -1.0, 2.0, map_i(),
```

Comment:
Why did you change this?

Reply:
My hunch is that the original frustums had upper and lower bases that were very large relative to its height, causing the map shape to have very acute/obtuse angles and probably contributing to the bulged map numerical inverse failing for extreme cases. Making the frustum a little taller improves this.

The ratios of the lengths/bases/heights tested still cover a broad range, and the frustums used in the binarycompactobject domain fall comfortably in this range so I feel it's fine to restrict the range of frustums that are tested.

I can add a comment saying that "empirically, bulged frustums should have a ratio of base to height that is no greater than 6:1, at higher ratios the root finding begins to fail" or something like that